### PR TITLE
add installer

### PR DESCRIPTION
**Pull request by pynixwang**

### CODE REVIEW

File: lib/vundle_cli/finder.rb [hunk 1]
```diff
@@ -27,6 +27,7 @@ def list
 
     def find
       say "Searching..."
+      found=false
```
Comment (baopham) [inline lib/vundle_cli/finder.rb]: can you add spaces around `=`? thanks

**Comment by baopham** [comment]
thanks, will test this weekend

File: lib/vundle_cli/finder.rb (hunk 1 above)
Comment (pynixwang) [inline lib/vundle_cli/finder.rb]: sorry，I will fix it.

File: lib/vundle_cli/installer.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        open(@vimrc, 'r').each { |l| 
+          if l.chomp =~ /vundle#begin/
+            l<<"\nPlugin '#{@plugin}'"
+          end
+          tmp << l
+        }
+        tmp.close
+        FileUtils.mv(tmp.path, @vimrc)
+        say "Waiting for plugin installing complete..."
+        `vim +PluginInstall +qall`
```
Comment (baopham) [inline lib/vundle_cli/installer.rb]: lets use `system "vim +PluginInstall +qall"` instead?

Comment (pynixwang) [inline lib/vundle_cli/installer.rb]: I really do not know what is the difference.

Comment (pynixwang) [inline lib/vundle_cli/installer.rb]: with system we can see the progress.

Comment (baopham) [inline lib/vundle_cli/installer.rb]: yes, as a user, I'd want to see the progress, so a little bit more user friendly

**Pull request merged**